services/src/main/java/org/keycloak/executors/DefaultExecutorsProviderFactory.java

```diff
@@ -100,6 +103,24 @@ public String getId() {
         return "default";
     }
 
+    @Override
+    public List<ProviderConfigProperty> getConfigMetadata() {
+        return ProviderConfigurationBuilder.create()
+                .property()
+                .name("<task-type>-min")
+                .type("int")
+                .helpText("The number of threads to keep in the pool, even if they are idle.")
+                .defaultValue(DEFAULT_MIN_THREADS)
+                .add()
+                .property()
+                .name("<task-type>-max")
+                .type("int")
+                .helpText("The maximum number of threads to allow in the pool.")
+                .defaultValue(DEFAULT_MAX_THREADS)
+                .add()
+                .build();
+    }
```

Comment:
Wonder if this is useful at all as the `task types` are not defined anywhere. This whole thing is a bit messy to be honest, and also quite crazy that we have potentially up to 64 x <several types> threads running in the background doing background jobs. Honestly got no clue why we even have multiple different pools for background tasks.

Perhaps just leave this one out for now?